server/src/core/auth/services/token.service.ts

```diff
@@ -86,6 +86,36 @@ export class TokenService {
     };
   }
 
+  async generateApiKeyToken(
+    workspaceId: string,
+    apiKeyId: string,
+    expiresAt?: Date | string,
+  ): Promise<AuthToken> {
+    const secret = this.environmentService.getLoginTokenSecret();
```

Comment:
@martmull We shouldn't use the same secret as the login token, but instead we should create a new one it's more secure.

Reply:
ok